src/lib/tabs/tab-nav-bar/tab-nav-bar.html

```diff
@@ -0,0 +1,4 @@
+<div class="md-tab-header">
```

Comment:
Thanks for the input Marcy. Jeremy and I discussed this and agree that we can move this responsibility to the user by making this a component with an attribute selector.

Usage of this component will be written by the user as such:
`<nav md-tab-nav-bar aria-label="navigation links"> ... </nav>`